.github/workflows/quality.yml

```
@@ -0,0 +1,203 @@
+          gradle-version: wrapper
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Run Spotless Check
+        run: ./gradlew spotlessCheck --no-daemon
+
+  security-scanning:
+    name: Security Vulnerability Scan
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)